extensions/notebook/package.json

```diff
@@ -282,6 +304,12 @@
         {
           "command": "notebook.command.searchUntitledBook",
           "when": "false"
+        },
+        {
+          "command": "notebook.command.trustBook"
```

Comment:
This needs to have a when clause to only show when on a book tree item :

`"when": "view == bookTreeView && viewItem == savedBook`

Reply:
Nice. Thanks!